prov/efa/src/rxr/rxr_pkt_type_req.h

```diff
@@ -128,6 +129,18 @@ struct rxr_rtm_base_hdr *rxr_get_rtm_base_hdr(void *pkt)
 	return (struct rxr_rtm_base_hdr *)pkt;
 }
 
+struct rxr_dc_eager_medium_rtm_base_hdr {
```

Comment:
dc_eager_medium_rtm is too confusing. Let's  just have separate types for eager and medium.

Reply:
Struct name changed, resolving.